cli/config/configfile/file.go

```diff
@@ -43,6 +43,7 @@ type ConfigFile struct {
 	NodesFormat          string                      `json:"nodesFormat,omitempty"`
 	PruneFilters         []string                    `json:"pruneFilters,omitempty"`
 	Proxies              map[string]ProxyConfig      `json:"proxies,omitempty"`
+	InsecureRegistries   []string                    `json:"insecure-registries,omitempty"`
```

Comment:
Yes, that's why I added it locally (after consulting with @estesp). If we could get it from the engine that would be better for sure.